src/test/java/com/aws/greengrass/clientdevices/auth/helpers/CertificateTestHelpers.java

```diff
@@ -51,46 +69,112 @@ private CertificateTestHelpers() {
         Security.addProvider(new BouncyCastleProvider());
     }
 
+    private enum CertificateTypes {
```

Comment:
All this fuzz here is just adding helpers to create correctly generated root cas, intermediate cas, and server certificates. You could verify it generates correct certs using `openssl verify`